### ISSUE TITLE
False positive `OneLineBetweenClassVisibilityChanges` lint on properties near comments

### ISSUE THREAD

**Issue by bakerkretzmar** [tighten/tlint#262]
The class below triggers the lint on the `public string $class` line. If I remove the comment lines above that line the lint doesn't show, so it's probably something to do with the comments.

```php
<?php

namespace App\Enums;

use App\Columns\IntegerColumn;
use App\Columns\StringColumn;
use BenSampo\Enum\Enum;
use InvalidArgumentException;

final class FieldType extends Enum
{
    const STRING = 'string';
    // const TEXT = 'text';
    const INTEGER = 'integer';
    // const DECIMAL = 'decimal';
    // const DATE = 'date';
    // const DATETIME = 'datetime';

    public string $class;

    public function __construct($enumValue)
    {
        parent::__construct($enumValue);

        $this->class = match ($enumValue) {
            self::STRING => StringColumn::class,
            // self::TEXT => StringColumn::class,
            self::INTEGER => IntegerColumn::class,
            // self::DECIMAL => StringColumn::class,
            // self::DATE => StringColumn::class,
            // self::DATETIME => StringColumn::class,
            default => throw new InvalidArgumentException("Unknown FieldType: '{$enumValue}'"),
        };
    }
}
```
